### ISSUE TITLE
After paste a pic into the editor, every ENTER input will make the editor flashing

### ISSUE THREAD

**Issue by Shawlaw** [marktext/marktext#2597]
### Description

version: 0.16.3

- [x] Can you reproduce the issue? 

### Steps to reproduce

1. New a markdown file
2. Paste a pic in to the editor
3. Try input any ENTER after the pic inside editor

**Expected behavior:**

UI doesn't flash 

**Actual behavior:**

UI flashes on every ENTER input

**Link to an example: [optional]**

Here is the screen-record gif

![ui-flashing](https://user-images.githubusercontent.com/8605553/120910341-a6358080-c6b0-11eb-8441-770c019f7fa2.gif)

### Versions

- Mark Text version: v0.16.3
- Operating system: Windows 10 20H2

**Comment by chikazin** [comment]
My computer can reproduce the same issue.
And I found that once the content I wrote exceeded the default height of UI (`scroll bar` appearing consequently), the UI flashing issue happened when typing at the bottom of UI. 

Here is the screen-record gif

![issue](https://user-images.githubusercontent.com/44538971/143666357-039acb89-942b-4515-b395-3a5c1eea0ad9.gif)

In fact, the flicker of UI is more serious than that captured by gif.
